src/data-access/FetchSearchEntityRepository.ts

```diff
@@ -47,6 +47,48 @@ export default class FetchSearchEntityRepository implements SearchEntityReposito
 
 		const data = await response.json();
 
+		// Inject data types
```

Comment:
But the datatype is already included in the response?

See for example https://www.wikidata.org/w/api.php?action=wbsearchentities&search=commons&language=en&type=property

Reply:
How on earth I missed it? *Headdesks*